package.json

```diff
@@ -0,0 +1,61 @@
+{
+  "name": "@financialforcedev/orizuru-transport-redis",
+  "version": "3.0.3",
```

Comment:
Version number seems high for a new package. Perhaps it should be 0.something?

Reply:
Yes!